packages/apps/kubernetes/templates/csi/deploy.yaml

```diff
@@ -16,12 +16,10 @@ spec:
     spec:
       serviceAccountName: {{ .Release.Name }}-kcsi
       priorityClassName: system-cluster-critical
-      nodeSelector:
-        node-role.kubernetes.io/control-plane: ""
       tolerations:
         - key: CriticalAddonsOnly
           operator: Exists
-        - key: node-role.kubernetes.io/master
```

Comment:
it is better to leave both... as you may run Cozy on relatively old k8s versions.

Reply:
We do not wish to support such environments.

Overall, setting correct tolerations according to the environments is a huge amount of work which should be done for all charts in the repository, not only for managed Kubernetes. Let's leave it for the next release